promql/parse.go

```diff
@@ -447,7 +447,7 @@ func (p *parser) expr() Expr {
 		vecMatching := &VectorMatching{
 			Card: CardOneToOne,
 		}
-		if op == itemLAND || op == itemLOR {
```

Comment:
With three symbols already, this should be worth adding `isSetOperator`

https://github.com/prometheus/prometheus/blob/master/promql/lex.go#L51-L72